packages/plugin-ext/src/common/plugin-api-rpc.ts

```diff
@@ -163,7 +163,7 @@ export const emptyPlugin: Plugin = {
 };
 
 export interface PluginManagerExt {
-    $stopPlugin(contextPath: string): PromiseLike<void>;
+    $stop(pluginId?: string): PromiseLike<void>;
```

Comment:
why break the name of this method ?

Reply:
Because it does not do what it means: It does not stop one plugin but all plugins. `contextPath` is unused, changing `contextPath` to optional `pluginId` to stop either one plugin or all was already a breaking change. I could introduce another method, but then `stopPlugin` won't be used anywhere in the Theia repo. I did not see a reason why to keep an unused method.